PixiEditor/Helpers/Converters/FileExtensionToColorConverter.cs

```diff
@@ -7,30 +11,34 @@ namespace PixiEditor.Helpers.Converters
     public class FileExtensionToColorConverter :
         SingleInstanceConverter<FileExtensionToColorConverter>
     {
-        private static readonly SolidColorBrush PixiBrush = ColorBrush(226, 1, 45);
+        private static readonly Dictionary<string, SolidColorBrush> extensions2Brushes;
 
-        private static readonly SolidColorBrush PngBrush = ColorBrush(56, 108, 254);
+        public static readonly SolidColorBrush UnknownBrush = ColorBrush(100, 100, 100);
 
-        private static readonly SolidColorBrush JpgBrush = ColorBrush(36, 179, 66);
-
-        private static readonly SolidColorBrush UnknownBrush = ColorBrush(100, 100, 100);
+        static FileExtensionToColorConverter()
+        {
+            extensions2Brushes = new Dictionary<string, SolidColorBrush>();
+            AssignFormat2Brush(Constants.NativeExtension, ColorBrush(226, 1, 45));
+            AssignFormat2Brush(ImageFormat.Png, ColorBrush(56, 108, 254));
+            AssignFormat2Brush(ImageFormat.Jpeg, ColorBrush(36, 179, 66));
+            AssignFormat2Brush(ImageFormat.Bmp, ColorBrush(40, 170, 236));
```

Comment:
And I think a orange would fit better here as it's hard to distinguish from the png color

Reply:
> According to the [Wikipedia](https://en.wikipedia.org/wiki/BMP_file_format).
> 
> > The BMP file format is capable of storing digital images both monochrome and color, in various color depths, and optionally with data compression, alpha channels, and color profiles.

I did research and could not find an example/API that could allow me to do that. I think having transparency in BMP is not a critical feature.

Reply:
orange  for which extension ?

Reply:
Bmp

Reply:
Done